queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT name, date_of_birth FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name, neutered, escape_attempts FROM animals WHERE neutered = 'TRUE' AND escape_attempts < 3;
+SELECT name, date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts, weight_kg FROM animals WHERE weight_kg > 10.5;
+SELECT name, neutered FROM animals WHERE neutered = 'TRUE';
+SELECT name FROM animals WHERE NOT name = 'Gabumon';
+SELECT name, weight_kg FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] For the 1st, 6th, 7th, and 8th solutions we have to find all the animals and not only the names. I would recommend using * instead of name for those solutions.

Reply:
Fixed